inst/extdata/ontomap.tsv

```diff
@@ -1,41 +1,33 @@
-dataset_name	original_column_name	original_cell_name	ontology_ID	ontology_cell_name	function_name
-macrophage_differentiation_protein_SCoPE2	celltype	Macrophage	CL:0000235	macrophage	SCoPE2
-macrophage_differentiation_protein_SCoPE2	celltype	Monocyte	CLO:0009465	U-937 cell	SCoPE2
-mouse_gastrulation_scNMT	lineage	Epiblast	NA	NA	scNMT
-mouse_gastrulation_scNMT	lineage	Mesoderm	NA	NA	scNMT
-mouse_gastrulation_scNMT	lineage	Primitive_Streak	NA	NA	scNMT
-mouse_gastrulation_scNMT	lineage	Ectoderm	NA	NA	scNMT
-mouse_gastrulation_scNMT	lineage	Endoderm	NA	NA	scNMT
-mouse_gastrulation_scNMT	lineage	ExE_ectoderm	NA	NA	scNMT
-mouse_gastrulation_scNMT	lineage	Visceral_endoderm	NA	NA	scNMT
-mouse_gastrulation_scNMT	lineage	Primitive_endoderm	NA	NA	scNMT
-mouse_visual_cortex_scRNAseq	class	Astrocyte	NA	NA	CITEseq
-mouse_visual_cortex_scRNAseq	class	GABA-ergic Neuron	NA	NA	CITEseq
-mouse_visual_cortex_scRNAseq	class	Endothelial Cell	NA	NA	CITEseq
-mouse_visual_cortex_scRNAseq	class	Glutamatergic Neuron	NA	NA	CITEseq
-mouse_visual_cortex_scRNAseq	class	Microglia	NA	NA	CITEseq
-mouse_visual_cortex_scRNAseq	class	Oligodendrocyte.3	NA	NA	CITEseq
-mouse_visual_cortex_scRNAseq	class	Oligodendrocyte.2	NA	NA	CITEseq
-mouse_visual_cortex_scRNAseq	class	Oligodendrocyte.1	NA	NA	CITEseq
```

Comment:
Thanks @LiNk-NY  for noticing this.

Yes, I just checked the cell types of scRNAseq (which is part of the seqFISH dataset) and I removed them because they are just duplicates.